docs/src/charm/howto/openstack.md

```diff
@@ -0,0 +1,120 @@
+# Integrating with OpenStack
+
+This guide explains how to integrate {{product}} with the OpenStack cloud
+platform.
+
+## Prerequisites
+
+To follow this guide, you will need:
+
+- An [OpenStack][openstack] cloud environment.
+- Juju set-up as the deployment tool for Openstack. Please refer to
+  [Juju and Openstack][juju-openstack] documentation for more information.
+- A Juju [controller][controller] with [access][credentials] to the OpenStack 
+  cloud environment.
+- A Juju [model][model] for deploying {{product}} on OpenStack.
```

Comment:
Once you have bootstrapped a controller on openstack -- it's easy to make a new model for your k8s cluster